data-processing-lib/python/src/data_processing/data_access/data_access_factory.py

```diff
@@ -86,13 +82,36 @@ def __add_data_navigation_params(self, parser):
                 "s3-path/your-output-bucket",
                 "Path to output folder of processed files",
             ],
+            "da_class": [
+                "package[.module].classname",
+                "Class name that implements the desired data access",
+            ],
+            "other attributes": [
```

Comment:
better to not have a space for key value while other use `_` . Also, rather than "other attributes", would be better to be more specific, such as "da_class_parameters"

Reply:
one suggestion is to define as kwargs, which are specific to the selected data access class (ex: "output_type")
```suggestion
            "kwargs": [
```